### ISSUE TITLE
On qtcreator install sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by vasiliyaltunin** [msys2/MINGW-packages#7528]
On fresh install today i cant install  mingw-w64-x86_64-qt-creator  

```
$ pacman -S  mingw-w64-x86_64-qt-creator
warning: mingw-w64-x86_64-qt-creator-4.14.0-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (1) mingw-w64-x86_64-qt-creator-4.14.0-1

Total Installed Size:  117.42 MiB
Net Upgrade Size:        0.00 MiB

:: Proceed with installation? [Y/n] y
(1/1) checking keys in keyring                                                                                                   [#############################################################################] 100%
(1/1) checking package integrity                                                                                                 [#############################################################################] 100%
(1/1) loading package files                                                                                                      [#############################################################################] 100%
(1/1) checking for file conflicts                                                                                                [#############################################################################] 100%
(1/1) checking available disk space                                                                                              [#############################################################################] 100%
:: Processing package changes...
(1/1) reinstalling mingw-w64-x86_64-qt-creator                                                                                   [#############################################################################] 100%
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
C:/msys64/mingw64/bin/sdktool.exe: error while loading shared libraries: ?: cannot open shared object file: No such file or directory
```
It seems qt-creator depend on ICU-67, but ICU-68 installed!

**Comment by lazka** [comment]
ok, I think it's bash.

**Comment by Alexpux** [comment]
@lazka it builds fine locally for me

**Comment by lazka** [comment]
ok, can you share the builds via yandex disk again?

qmltyperegistrar.exe errors out here every time.

**Comment by lazka** [comment]
The error seems to be due to path conversion, `MSYS2_ARG_CONV_EXCL='*' fixes it`

**Comment by lazka** [comment]
@Alexpux it worked, I sent them to David

**Comment by lazka** [comment]
Duplicate of #7526
